docs/devops-guide/quickstart.md

```diff
@@ -127,6 +127,18 @@ Check the firewall status with:
 sudo ufw status verbose
 ```
 
+If you are using `nftables`, you can add the following line to `/etc/nftables.conf` in the "input" chain, which is in the "inet filter" table:
+
+```
+tcp dport { 80, 443, 10000, 22, 3478, 5349 } ct state new accept
```

Comment:
Port 5349 is no longer necessary, but 5222 is.

Reply:
TCP port 5349 is in the documentation.  Shall I update the documentation too (changing port 5349 to 5222)?

Reply:
What is this about? 5349 is the default turns port.
https://github.com/jitsi/jitsi-meet/blob/8604b1f64d959b652d98d76c99c1140e1bb16e62/doc/debian/jitsi-meet-turn/turnserver.conf#L14

Reply:
Ops, my bad I read too quickly and mixed it up with the XMPP component port.

Reply:
We still need 5222 though right?

Reply:
From the outside no, it is used only from jvb, jibri and maybe jigasi(jiigasi can use bosh though and port 443), and normally that will be only the internal network for prosody.

Reply:
Generally speaking yeah but if one needs to scale to multiple JVBs they would so I think we better mention it early WDYT?